torch/testing/_internal/distributed/rpc/rpc_agent_test_fixture.py

```diff
@@ -5,6 +5,10 @@
 
 
 class RpcAgentTestFixture(ABC):
+    @property
+    def name(self):
```

Comment:
we can use the existing API:

https://github.com/pytorch/pytorch/blob/f0e72e54cc3049e986adc5bff314dd0b649c780b/torch/testing/_internal/dist_utils.py#L194

Reply:
addressed